tests/generic_views/test_edit.py

```diff
@@ -90,6 +90,21 @@ def test_post_data(self):
         self.assertRedirects(res, '/list/authors/')
 
 
+@override_settings(ROOT_URLCONF='generic_views.urls')
```

Comment:
1. I think you just need a `urlpatterns` variable at the module level.
2. Maybe a simpler test could just call `get_context_data()` on an instantiated version of the mixin or one of its subclasses and verify 'form' is there.